starter/schemaorg/README.md

```diff
@@ -0,0 +1,140 @@
+# Starter Code for Schema2QA
+
+This directory contains the basic starter code to train a single-sentence
+Q\&A semantic parsing model from a Schema.org schema.
+
+## Installation
+The starter code requires`nodejs` (>=10.0) and `yarn` as a package manager. 
+See [nodejs](https://nodejs.org/en/download/) and [yarn](https://classic.yarnpkg.com/en/docs/install/) for installation details. 
+You can check your installation by running `node --version` and `yarn --version`.
+
+In addition, you will need two libraries from OVAL: 
+[genie-toolkit](https://github.com/stanford-oval/genie-toolkit)
+and [thingpedia-cli](https://github.com/stanford-oval/thingpedia-cli).
+`genie-toolkit` provides you tools to process schema.org data, generate manifest, and synthesize data. 
+`thingpedia-cli` provides easy way to download data from and upload data to Thingpedia. 
+
+Run the following command to install them: 
+```bash
+# install genie-toolkit
+git clone https://github.com/stanford-oval/genie-toolkit.git
+cd genie-toolkit
+yarn
+
+# install thingpedia-cli 
+yarn global add thingpedia-cli
+```
+
+After installation, you should get a command called `thingpedia`.
+If encounter `command not found`, make sure the Yarn global bin directory
+(usually `~/.yarn/bin`) is in your PATH. You can find the path with the command
+`yarn global bin`.
+
+```bash
+export PATH=~/.yarn/bin:$PATH
+```
```

Comment:
This installation bit doesn't seem right. We already have installation instructions for Genie in doc/install.md, and also the starter is literally inside the Genie repository so one must have cloned the repository first.